### PR TITLE
Replace data tables with indexed db and fix linting errors

### LINKED ISSUE #16: Remove Anvil Data Tables Services
We used anvil's data tables for the hackathon as it was quick to setup and we know it well. However, that puts a server side component into the mix and we should replace it.

We can use indexed db as a convenience - it would probably be nice for anyone who has created a ballot to have their own ballots easily available - but we should also be able to pull ballot details from chain for any given id.

At the moment, we're not storing the ballot name in the contract, so we'd need to decide how to handle that. Same goes for the end time.

### PR DESCRIPTION
**Pull request by meatballs**
closes #16

### CODE REVIEW

File: client_code/forms/routes/__init__.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
-from . import ballot, vote
-from .Home import Home
+from . import ballot, vote  # noqa F401
```
Comment (theref) [inline client_code/forms/routes/__init__.py:1]: what is this?

Comment (meatballs) [reply client_code/forms/routes/__init__.py:1]: It's a comment that stops flake8 compatible linters from complaining about an unused import. F401 is the error code for that.

https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html

As it's just a comment, it has no effect on execution.

Comment (meatballs) [reply client_code/forms/routes/__init__.py:1]: I like to get linting errors to zero so I can tell when I've intoduced one!

**Comment by meatballs** [comment]
Should be good to look at again now

**Pull request merged**